### ISSUE TITLE
Within can not be used inside an argument for other steps due to ambiguous match

### ISSUE THREAD

**Issue by denzelem** [makandra/spreewald#53]
```
Then I should see "You will receive an email within the next few minutes. It contains instructions for changing your password."
```

```
Ambiguous match of "I should see "You will receive an email within the next few minutes. It contains instructions for changing your password."":

spreewald-1.2.6/lib/spreewald/web_steps.rb:41:in `/^(.*) within (.*[^:])$/'
spreewald-1.2.6/lib/spreewald/web_steps.rb:155:in `/^(?:|I )should see "([^"]*)"$/'

```

**Comment by henning-koch** [comment]
I believe this will be really hard to fix since:

- We need the possibility to have double quotes in the selector (`... within "foo bar"`)
- We can't count double quotes in a Regexp
- We can't "exclude" words in a Regexp

Gues this is a limitation of all meta-steps like `within`?

**Comment by foobear** [comment]
@triskweline Could this be fixed by making Spreewald's `within` step `overridable` using [cucumber_priority](https://github.com/makandra/cucumber_priority)?

**Comment by triskweline** [comment]
Probably, yes. This issue predates `cucumber_priority`.

Since all Spreewald steps (including `Then I should see`) are already `overridable`, you would need to give `within ...` a lower priority than the others.

**Comment by judithroth** [comment]
This is fixed in 1.12.4.
